sponsorship_compassion/models/account_invoice.py

```diff
@@ -53,13 +52,38 @@ def _compute_last_payment(self):
     @api.multi
     def _compute_invoice_type(self):
         for invoice in self.filtered(lambda i: i.state in ('open', 'paid')):
-            categories = invoice.with_context(lang='en_US').mapped(
-                'invoice_line_ids.product_id.categ_name')
-            if SPONSORSHIP_CATEGORY in categories:
+
+            sponsorship_cat = self.env.ref(
+                'sponsorship_compassion.product_category_sponsorship')
```

Comment:
To avoid error at module init, we use a default value if the xml reference is not yet loaded.
```suggestion
                'sponsorship_compassion.product_category_sponsorship', -1)
```